test/lstm.test.ts

```diff
@@ -18,7 +18,7 @@ describe('lstm', () => {
     const h = sm.full([1, 128], 0)
     const c = sm.full([1, 128], 0)
     const l = sm.module.lstm(64, 128)
-    const [hn, cn] = l(x, h, c)
+    const [hn] = l(x, h, c)
```

Comment:
hmm this should return two values

Reply:
Correct, the latter value was unused so I removed, but can just comment out the ESLint warning instead